libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/extensions/LocaleEx.kt

```diff
@@ -1,4 +1,4 @@
-package com.mapbox.navigation.base.extensions
+package com.mapbox.navigation.base.internal.extensions
```

Comment:
why do we need this internal?

Reply:
Same as above. All these extensions are only used internally and I don't think they're that useful for developers as they can create their own. The less `public` APIs, the better. Otherwise we need to maintain them, we cannot change them without breaking SemVer, etc.